executor/show.go

```diff
@@ -256,7 +256,28 @@ func (e *ShowExec) fetchShowBind() error {
 	} else {
 		bindRecords = domain.GetDomain(e.ctx).BindHandle().GetAllBindRecord()
 	}
+	// Remove the invalid bindRecord.
+	ind := 0
+	for _, bindData := range bindRecords {
+		if len(bindData.Bindings) > 0 {
```

Comment:
How could this happen?

Reply:
When the session binding is deleted, `len(bindData.Bindings)` maybe 0.